src/stash/src/postgres.rs

```diff
@@ -1169,11 +1189,28 @@ impl Consolidator {
         &mut self,
         id: Id,
         since: &Antichain<Timestamp>,
+        attempt: u64,
```

Comment:
Instead of providing `attempt` as an argument, what do you think about `priority`? Then the retry loop, which already tracks attempts, can adjust accordingly?

Reply:
I initially tried to put it there, but in the retry loop we only have access to an `Option<Client>`. It was much easier to put this in a place where we always have a `Client` available.